src/components/general/ProjectSingle.svelte

```diff
@@ -21,8 +21,8 @@
 	// Get optimal title image
 	let titleImage = $derived.by(() => {
 		if (!project?.titleImage?.[0]) {
-			// Fallback to first image if no titleImage
-			return project?.images?.[0] || null;
+			// Do not fallback to project.images[0] because it contains also images that should no longer be shown at all
+			return null;
```

Comment:
Have you tested what this looks like for projects without any title images?
It might make sense to add a css fallback to a solid colour background for projects without any title images, to keep the layout of the page intact. Unless there already is, then ignore what I said.